### ISSUE TITLE
pkg_resources.DistributionNotFound on v1.12.10

### ISSUE THREAD

**Issue by kyohei3** [googleapis/google-api-python-client#1670]
We are using bazel to build our app, and facing the following issue that is basically same with https://github.com/googleapis/google-api-python-client/issues/1622. 
I found #1622 was fixed by https://github.com/googleapis/google-api-python-client/pull/1623, but for some restriction on our environment, I cannot upgrade google-api-python-client to v2.x. 

May I create a pull request that apply the same fixes to the `v1` branch?

#### Environment details

Building a container image with Bazel python rules.

#### Code example

```python
from googleapiclient.model import JsonModel
```

#### Stack trace
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/app/googleapiclient/model.py", line 36, in <module>
    _LIBRARY_VERSION = pkg_resources.get_distribution("google-api-python-client").version
  File "/app/pkg_resources/__init__.py", line 466, in get_distribution
    dist = get_provider(dist)
  File "/app/pkg_resources/__init__.py", line 342, in get_provider
    return working_set.find(moduleOrReq) or require(str(moduleOrReq))[0]
  File "/app/pkg_resources/__init__.py", line 886, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/app/pkg_resources/__init__.py", line 772, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'google-api-python-client' distribution was not found and is required by the application
```

**Comment by parthea** [comment]
Hello @kyohei3 ,

Yes, PRs are welcome!

**Comment by parthea** [comment]
Fixed in #1677
